big_tests/tests/rdbms_SUITE.erl

```diff
@@ -431,6 +460,29 @@ test_restart_transaction_with_execute_eventually_passes(Config) ->
     called_times(3),
     ok.
 
+test_failed_transaction_with_execute_wrapped(Config) ->
+    % given
+    HostType = host_type(),
+    Pid = self(),
+    erase_table(Config),
+    prepare_insert_int8(Config),
+    ok = rpc(mim(), meck, new, [mongoose_rdbms_backend, [passthrough, no_link]]),
+    ok = rpc(mim(), meck, expect, [mongoose_rdbms_backend, execute, 4,
+                                   {error, simulated_db_error}]),
+    WrapperFun = fun(SqlExecute) ->
+        Pid ! msg_before,
+        Result = SqlExecute(),
+        Pid ! msg_after,
+        Result
+    end,
```

Comment:
Hmmm... how does this fail? I'm rather thinking what if the WrapperFun throws an exception, will it kill the process? Is it ok or not? @arcusfelis

Reply:
I wanted to test this code:

```
    WrapperFun = fun(SqlExecute) ->
        error(oopsie)
    end,
```

Inside a transaction and not inside a transaction.
We should check that the process is still running in this case.

This code tests something different.